### DISCUSSION TITLE
cpm metrics in timebucket

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#7769] [Q&A]
I have some fuzzy about cpm metrics in different dimensions time_bucket. When calculating the total requests during some time,
I calculate the sum for "total" field in metrics-cpm indice ,but I found the result are different when the total metric in the dimensions of minute time_bucket and  hour timebucket .

**Comment by wu-sheng** [comment]
There are discussion already. Read them first.
